crates/core/c8y_api/src/smartrest/smartrest_serializer.rs

```diff
@@ -219,76 +219,122 @@ mod tests {
 
     #[test]
```

Comment:
These tests should give a broad overview of the updated functionality. Mostly, it means we no longer have unnecessary quotes and newlines, and generating the messages is simpler.